### ISSUE TITLE
Background images are loading twice

### ISSUE THREAD

**Issue by ashehzadgec** [aFarkas/lazysizes#883]
⚠️ Search for existing open/closed issues/discussions before you open your issue.
Hi there

I noticed that my background images loading twice when cache is disabled and loads once when cache is enabled. Is it a normal behavior or what I am doing wrong. Below is attached pics for your reference.

![bg image loading once with cache enable](https://user-images.githubusercontent.com/65112198/113140286-a85b0900-9241-11eb-89fb-e062c578f28c.png)
![bg image loading twice wit cache disable](https://user-images.githubusercontent.com/65112198/113140293-a98c3600-9241-11eb-9b9c-96ab4e41f4ca.png)
![html for bg image](https://user-images.githubusercontent.com/65112198/113140297-aa24cc80-9241-11eb-9fcb-64866acf860f.png)

Regards,
Shehzad Asif

**Comment by aFarkas** [comment]
Yes, this is the normal behavior of bgset. Because it first loads it for `picture > img` and second for `style: background-image: ...`. 

If this is bothering you you can replace bgset in most cases with CSS object-fit.

**Comment by ashehzadgec** [comment]
thanks
